components/admin_console/admin_sidebar_header/admin_sidebar_header.tsx

```diff
@@ -13,12 +13,12 @@ import Avatar from 'components/widgets/users/avatar';
 
 import AdminNavbarDropdown from 'components/admin_console/admin_navbar_dropdown';
 
-export default class SidebarHeader extends React.Component {
-    static propTypes = {
-        currentUser: PropTypes.object,
-    }
+type State = {
```

Comment:
This is not the `State` this is the `Props` type, the `State` type makes sense for the internal state of the component, and in this case are the `Props` or the components (obtained by redux from the global state, but from the component perspective this is the `Props`.